manifests/init.pp

```diff
@@ -72,6 +72,7 @@
   $qpid_router_broker_port   = $capsule::params::qpid_router_broker_port,
 ) inherits capsule::params {
 
+  include ::apache
```

Comment:
For this change I suppose I can live with that -- I really need to bite the bullet and take the time to split this up more cleanly for the various concerns it tackles.